content/flux/v0/spec/options.md

```diff
@@ -62,4 +62,9 @@ option location = timezone.fixed(offset: -5h)
 option location = timezone.location(name: "America/Denver")
 ```
 
+{{% note %}}
+The `location` option affect the boundaries used for windowing, although the times may change, the timezone used in the `_time` results column will always be UTC.
+{{% /note %}}
```

Comment:
We're moving towards using [GitHub-style alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) instead of the `note` and `warn` shortcodes.

```suggestion
>  [!Note]
> The `location` option only affects boundaries used for windowing, specifically around time shifts
> like daylight savings. It does not change timestamps in the `_time` column, which are always UTC.
```